pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,50 @@ permalink: discover-nmap.html
 ---
 
 
+As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to [the other discovery methods](https://guides.tidalmg.com), you can use [Nmap](https://nmap.org/) to facilitate a part of the discovery process.
```

Comment:
Thanks for pointing it out. In this case, I do feel "As part of your could..." is appropriate. since we are referring to the importance to have something ("unquantifiable") without giving it measurable value.